views/layout.erb

```diff
@@ -76,6 +76,9 @@
     .jumbotron{
       margin-top: 28px;
     }
+    .alert{
+     margin:20px 0 0 0;   
+    }
```

Comment:
Both `.jumbotron` and `.alert` affect all of bootstrap styles. Why is this necessary? Are we breaking a bootstrap convention that requires us to modify the built in styles?